src/Service.Host/client/src/components/reports/SpendBySupplier.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useMemo } from 'react';
+import {
+    Loading,
+    ReportTable,
+    BackButton,
+    ExportButton,
+    reportSelectorHelpers
+} from '@linn-it/linn-form-components-library';
+import Grid from '@mui/material/Grid';
+import { useSelector, useDispatch } from 'react-redux';
+import queryString from 'query-string';
+import history from '../../history';
+import config from '../../config';
+import spendBySupplierActions from '../../actions/spendBySupplierActions';
+
+function SpendBySupplierReport() {
+    const options = useMemo(() => queryString.parse(window.location.search) || {}, []);
```

Comment:
Why useMemo()? (Not suggesting its wrong, just curious.. Never really used it before)

Reply:
Purely because I used it in the other reports, originally in stores likely copied from an Adam or someone else report.. might try one of our other new options we've mentioned? Even if just for a change